### PR TITLE
viewing products goes from product type to product list to detailed p…

### PR DESCRIPTION
**Pull request by mccordgh**
# Description
Per requirements, When choosing to view products you see in this order:
1. All Product Types (click through to...)
2. All Products of that Product Type (click through to...)
3. Details for that individual Product clicked and an Add to Order button if logged in.

# Related Ticket(s)
#34 #32 

## Steps to Test
1. pull down branch mm-products-view-triple
2. run server
3. navigate to products/
4. click a product type
5. click a product
6. verify detailed view appears

### CODE REVIEW

File: bangazonsite/initial_site/templates/initial_site/product_detail.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
-{% extends "base_generic.html" %}
```
Comment (asimonia) [inline bangazonsite/initial_site/templates/initial_site/product_detail.html]: {% extends "base_generic.html" %}
{% block content %}

File: bangazonsite/initial_site/templates/initial_site/product_detail.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-{% block content %}
```
Comment (asimonia) [inline bangazonsite/initial_site/templates/initial_site/product_detail.html]: {% block content %}_
{% endblock %}_

**Review by asimonia — COMMENTED** [review]
👍

**Review by nchemsak — APPROVED** [review]
💯

**Pull request merged**